horace_core/sqw/PixelData/@PixelData/get_pixels.m

```diff
@@ -0,0 +1,90 @@
+function pix_out = get_pixels(obj, pix_indices)
```

Comment:
For clarity should we rename these arguments `abs_pix_indices`, or those in relative-index calls `page_pix_indieces`